website/manual.md

```diff
@@ -312,44 +312,44 @@ file_server --reload
 
 ### Run subprocess
 
-```
+[API Reference](https://deno.land/typedoc/index.html#run)
+
+Example:
+
+```ts
 const p = Deno.run({
-    args: ["deno", "--allow-read", "https://deno.land/x/examples/cat.ts", "README.md"],
+  args: ["deno", "https://deno.land/welcome.ts"]
 });
 
 // start subprocess
 await p.status();
 ```
 
-When this program is started, the user is prompted for permission to run
-subprocess:
-
-```
-> deno https://deno.land/x/examples/subprocess_simple.ts
-⚠️  Deno requests access to run a subprocess. Grant? [yN] y
-```
-
-For security reasons, deno does not allow programs to run subprocess without
-explicit permission. To avoid the console prompt, use a command-line flag:
+Run it:
 
 ```
-> deno https://deno.land/x/examples/subprocess_simple.ts --allow-run
+> deno --allow-run https://deno.land/x/examples/subprocess_simple.ts
+Welcome to Deno 🦕
 ```
 
 By default when you use `deno.run()` subprocess inherits `stdin`, `stdout` and
 `stdout` of parent process. If you want to communicate with started subprocess
 you can use `"piped"` option.
 
-```
+```ts
 const decoder = new TextDecoder();
 
-const filesToCat = Deno.args.slice(1);
-const subprocessArgs = ["deno", "--allow-read", "https://deno.land/x/examples/cat.ts", ...filesToCat];
+const fileNames = Deno.args.slice(1);
 
 const p = Deno.run({
-  subprocessArgs,
+  args: [
+    "deno",
+    "--allow-read",
+    "https://deno.land/x/examples/cat.ts",
+    ...fileNames
+  ],
   stdout: "piped",
-  stderr: "piped",
+  stderr: "piped"
 });
 
 const { code } = await p.status();
```

Comment:
Use `p.output()` 

https://github.com/denoland/deno/blob/a5720d9e28f7ee5868049504816405392c2821d7/js/process_test.ts#L180-L187

Reply:
Sure, I think it'd be nice to have `await p.error()` as well.

Reply:
These examples aren't meant to be complete documentation, but rather jumping off points for further investigation.

`p.error()` for stderr? That's handled with `p.output()` if you set stderr to `piped`

Reply:
So if I set `stderr: piped` then error will be redirected to `p.stdout`? I got a bit confused because `output` does not read from `p.stderr`.

Reply:
Hmm maybe I'm wrong... yes it would be nice to have a way to buffer stderr.

Reply:
I'll open a separate PR for that.